tooling/nargo_cli/tests/execution_success/nested_array_dynamic/src/main.nr

```diff
@@ -30,7 +30,7 @@ fn main(mut x : [Foo; 4], y : pub Field) {
     } else {
         x[y].a = 100;
     }
-    assert(x[y].a == 50);
+    assert(x[3].a == 50);
```

Comment:
```suggestion
    assert(x[y].a == 50);
```

What's the reasoning behind changing this?

Reply:
I was thinking to do this change so that we guarantee that the dynamic index I change inside the if statement is actualy `3`. If `x[y].a` actually wrote to some other index asserting on `y` later would still hold.